src/Tizen.TV.UIControls.Forms/Renderer/MediaPlayerImpl.cs

```diff
@@ -20,41 +20,51 @@
 using System.Runtime.CompilerServices;
 using System.Threading.Tasks;
 using Tizen.Multimedia;
+using Tizen.TV.Multimedia;
 using Tizen.TV.UIControls.Forms.Renderer;
 using Xamarin.Forms;
 using Xamarin.Forms.Internals;
 using Xamarin.Forms.Platform.Tizen;
-
+using TVM = Tizen.TV.Multimedia;
 [assembly: Xamarin.Forms.Dependency(typeof(MediaPlayerImpl))]
 namespace Tizen.TV.UIControls.Forms.Renderer
 {
     public class MediaPlayerImpl : IPlatformMediaPlayer
     {
-        Player _player;
-
-        bool _cancelToStart;
-        DisplayAspectMode _aspectMode = DisplayAspectMode.AspectFit;
-        Task _taskPrepare;
-        TaskCompletionSource<bool> _tcsForStreamInfo;
-        IVideoOutput _videoOutput;
+        TVM.Player _player;
+        Lazy<DRMManager> _drmManager = new Lazy<DRMManager>(DRMManager.CreateDRMManager(TVM.DRMType.Playready));

```

Comment:
Acutally, DRMManager was created when Lazy class was created, because you pass DRMManager instance as constructor paramater